image-tools-interface.md

```diff
@@ -0,0 +1,52 @@
+# Image Tools Interface
+
+The below document describes an interface for the OCI Image Tools project as desired by the OCI Certification Program working group (Cert WG.)
+
+Goals:
+* An engineer can verify a container image complies with the OCI Image Spec (https://github.com/opencontainers/image-spec/releases) 
+
+Inputs:
+* (REQUIRED): Path or reference to a container image
```

Comment:
If distribution is out of scope of image-spec, what would a "reference" be? A `https://` URL to the OCI Image Layout directory? And the software would download `${URL}/index.json` and other files as required by the [image layout content](https://github.com/opencontainers/image-spec/blob/master/image-layout.md#content)?